src/Nri/Colors.elm

```diff
@@ -172,32 +172,32 @@ blue =
 
 {-|
 
-<p style="font-size:2em; color: #00488A">#00488A</p>
+<p style="font-size:2em; color: #0e5595">#0e5595</p>
```

Comment:
Note: colors should get added to the StyleGuide in the main repo:  https://staging.noredink.com/style_guide#Nri/Colors.elm  (not sure if it's necessary to have the examples here in the docs because they're now in the style guide?)